ceph-collect

```diff
@@ -279,20 +290,32 @@ def collect_ceph_information(r, ceph_config, output_directory, timeout,
         files['fsid'] = bytes(r.get_fsid() + '\n', 'utf-8')
         files['ceph.conf'] = filter_config(
                 get_ceph_config(ceph_config), 
-                'plain'
+                'plain',
+                True
         )
     else:
         files['fsid'] = str(r.get_fsid()) + '\n'
         files['ceph.conf'] = str(
             filter_config(
                 get_ceph_config(ceph_config), 
-                'plain'
+                'plain',
+                True
             )
         )
     files['config'] = filter_config(
         ceph_mon_command(r, 'config dump', timeout, output_format),
-        output_format
+        output_format,
+        False
     )
+    if log_config: 
```

Comment:
While this is nice, it's not strictly needed to fix the issue. Let's keep ceph-collect simple and not extend it with command line options. If you want to know what's in ceph.conf, you can just extract it from the tarball.

Reply:
I think you need it if you want to filter your config: save it and extract the data has two problems, 1 is boring and 2 you need to know which file is. 
We implement this to allow people to filter the data. If they need to extract the data, they can filter it by themself.